test/prototype/test_low_bit_optim.py

```diff
@@ -372,10 +377,56 @@ def test_optim_bf16_stochastic_round_correctness(self):
                 loss1, loss2, msg=lambda msg: f"Iteration {idx}. {msg}"
             )
 
+    @pytest.mark.skipif(not TORCH_VERSION_AT_LEAST_2_3, reason="requires PyTorch >= 2.3")
```

Comment:
In our CI, min PyTorch version is 2.3. We don't need to check >=2.3 anymore. You can remove this line